docs/framework/configure-apps/file-schema/winforms/windows-forms-add-configuration-element.md

```diff
@@ -91,11 +92,11 @@ High DPI awareness is an opt-in feature; by default, the value of `DpiAwareness`
 The single default opt-in setting provided by setting `DpiAwareness` to `PerMonitorV2` is generally adequate for new Windows Forms applications. However, You can then opt out of individual high DPI improvements by adding the corresponding key to the application configuration file. For example, to take advantage of all the new DPI featuers except for dynamic DPI support, you would add the following to your application configuration file:
 
 ```xml
-<System.Windows.Forms.ApplicationConfigurationSection>
-  <add key="DpiAwareness" value="PerMonitorV2" />
-  <!-- Disable dynamic DPI support -->
-  <add key="EnableWindowsFormsHighDpiAutoResizing" value"false" />
-</System.Windows.Forms.ApplicationConfigurationSection>
+<System.Windows.Forms.ConfigurationSection>
```

Comment:
The original is correct in both cases (ApplicationConfigurationSection, not ConfigurationSection)